cpp/src/core/column.cu

```diff
@@ -236,12 +236,17 @@ legate::LogicalArray from_arrow(std::shared_ptr<arrow::Array> arrow_array)
   }
   auto array = runtime->create_array({std::uint64_t(arrow_array->length())},
                                      to_legate_type(arrow_array->type_id()),
-                                     1,
-                                     arrow_array->null_bitmap_data() != nullptr);
+                                     arrow_array->null_count() > 0,
```

Comment:
Let's try this! (I am slightly unsure if null count 0 and nullable are subtly different, but columns should be immutable, so...)